app.rb

```diff
@@ -0,0 +1,142 @@
+require 'date'
+require_relative 'library/classes/person'
+require_relative 'library/classes/teacher'
+require_relative 'library/classes/student'
+require_relative 'library/classes/book'
+require_relative 'library/classes/rental'
+
+# This class represents the main application for managing books, people, and rentals.
+
+# rubocop:disable Metrics
+class App
+  def display_menu
+    puts 'Please select an option by entering a number'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a Book'
+    puts '5. Create a Rental'
+    puts '6. List all rentals for a given person id'
+    puts '7. Exit'
+  end
```

Comment:
- [ ] This part should be in a file named `main.rb`. It should be an entry point of your app, where you call the `app.rb` file to start your application. Kindly make adjustments based on the requirements below 👍 

### Requirements:
![image](https://github.com/diarisdiakite/oop_school-library/assets/50990925/41258429-0ee8-4a48-b1d3-73402ff99690)

### Sample code snippet:
```
# main.rb
def main
  app = App.new()

  response = nil


  # *Present the user with a list of options to perform.
  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
       call function.....
    when "2"
       call function.....
        .
        .
        .
   
    end

    puts "\n"
  end
end

main()
```